devel/olm/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+name                olm
+version             2.3.0
+categories          devel security
+platforms           darwin
+maintainers         {@scarface-one}
+description         An implementation of the Double Ratchet cryptographic ratchet in C++
+long_description    An implementation of the Double Ratchet cryptographic ratchet described by \
+                    https://whispersystems.org/docs/specifications/doubleratchet/, written in C \
+                    and C++11 and exposed as a C API.
+
+license             Apache-2
+homepage            http://git.matrix.org/git/olm/about/
+
+master_sites        http://git.matrix.org/git/olm/snapshot
+distfiles           olm-${version}.tar.gz
```

Comment:
In general, we do not set `distfiles` directly, it is best practice to set `distname`, `worksrcdir` and `${extract.suffix}` separately as needed. For instance,  `${extract.suffix}` is `.tar.gz` by default. Actually, it looks like it does not need to be set at all because that appears to be the default.